### PR TITLE
style: refactor + fix style issues on welcome screen

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-5796

### CODE REVIEW

**Comment by Addminus** [comment]
image
on galaxy fold the welcome screen looks like this. iphone 12 pro is similar

**Comment by chybisov** [comment]
@Addminus tested the latest link on iPhone 14 Pro Max and looks fine in both Safari and Chrome. I'm curious where did you have a chance to test on Galaxy Fold? 😄
telegram-cloud-photo-size-2-5399846503775850202-y

**Comment by KriptoQraf** [comment]
testnet?

**Review by Addminus — APPROVED** [review]

**Review by chybisov — CHANGES_REQUESTED** [review]

File: packages/dapp/src/components/Widget/Widget.style.ts [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+
+export const WidgetWrapper = styled(Box, {
+  shouldForwardProp: (prop) => prop !== 'welcomeScreenClosed',
+})<WidgetWrapperProps>(({ theme, welcomeScreenClosed }) => ({
   minWidth: '392px',
-  margin: ' 0 auto',
   position: 'relative',
+  margin: ' 0 auto',
+
+  [`@media screen and (min-height: 900px)`]: {},
```
[the remark this answers is not in the document]
Comment (dennyscode) [reply packages/dapp/src/components/Widget/Widget.style.ts]: Thanks Eugene! Even found a few more of these! :crossed_swords:

**Comment by chybisov** [comment]
Small issue

https://github.com/lifinance/jumper.exchange/assets/18644653/cb2bf6a1-c743-4b5d-829e-efa5ce5986b3

File: packages/dapp/src/components/Widgets/Widgets.style.tsx [hunk 2]
```diff
@@ -5,23 +5,38 @@ export interface WidgetContainerProps extends Omit<BoxProps, 'component'> {
   welcomeScreenClosed: boolean;
 }
 
+export const hoverOffset = '24px';
```
Comment (chybisov) [inline packages/dapp/src/components/Widgets/Widgets.style.tsx:8]: Where is this used? 🙂

Comment (dennyscode) [reply packages/dapp/src/components/Widgets/Widgets.style.tsx:8]: It's removed. Good find!!

**Review by chybisov — APPROVED** [review]

**Pull request merged**